diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/chapter2/kdoc/KdocComments.kt

```diff
@@ -228,13 +221,13 @@ class KdocComments(configRules: List<RulesConfig>) : DiktatRule(
     private fun replaceWrongTagInClassKdoc(
         kdocBeforeClass: ASTNode,
         parameterName: String,
-        isParamTagNeeded: Boolean,
-        isFirstTagInKdoc: Boolean
+        isParamTagNeeded: Boolean
     ) {
-        val wrongTagText = if (isParamTagNeeded) "* @property $parameterName" else "* @param $parameterName"
-        val replaceText = if (isParamTagNeeded) "* @param $parameterName" else "* @property $parameterName"
+        val paramOrPropertySwitchText = if (isParamTagNeeded) "@property" to "@param" else "@param" to "@property"
```

Comment:
I see this code in 3+ places: can you please rework it as common functions?

```
fun replace(isParamTagNeeded, parameterName) {
    val (wrongTag, replacementTag) = if(isParamTagNeeded) (property, param) else (param, property)
    val wrongTagText = ...
    val replaceText = ...
    changeTagInKdoc(kdocBeforeClass, wrongTagText, replaceText)
}

```

Reply:
Done